src/site/content/en/blog/embed-best-practices/index.md

```diff
@@ -0,0 +1,314 @@
+      </tr>
+    </tbody>
+    <caption>Embed vs Facade.</caption>
+  </table>
+</div>
+
+A [facade](https://web.dev/third-party-facades/) is a static element that looks similar to the actual embedded third-party but is not functional and, therefore, much less taxing on the page load. Following are a few strategies to load such embeds optimally while still providing some value to the user.
+
+#### Use static images as facades
+
+Static images can be used instead of map embeds where you might not need to make the map interactive. You can zoom in on the area of interest on the map, capture an image, and use this instead of the interactive map embed. You can also use _DevTools **Capture node screenshot** _functionality_ _to capture a screenshot of the embedded `iframe` element, as shown below.
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/EJvMAEUmF3QNUDGBgfNR.png", alt="Capture node screenshot", width="400", height="500" %}
+
+DevTools captures the image as a  `png`, but you can also consider converting it to <code>[WebP format for better performance](https://web.dev/serve-images-webp/)</code>.
+
+
+#### Use dynamic images as facades
+
+This technique allows you to generate images corresponding to an interactive embed at run time. Following are some of the tools that allow you to generate static versions of embeds on your pages.
+
++ Maps Static API
+    The Google [Maps Static API](https://developers.google.com/maps/documentation/maps-static/overview) service generates a map based on the URL parameters included in a standard HTTP request and returns the map as an image you can display on your web page. The URL needs to include the Google Maps API key and must be placed in the `<img>` tag on the page as the `src` attribute. 
+
+    The [Static map maker](https://staticmapmaker.com/google/) tool helps to configure the parameters required for the URL and gives you the code for the image element in real-time.
+
+    The following snippet shows code for an image with the source set to a Maps Static API URL. It has been included in a link tag that ensures that the actual map can be accessed by clicking on the image. (Note: API key attribute is not included in the url)
+
+    ```html
+    <a href="https://www.google.com/maps/place/Albany,+NY/">
+    <img src="https://maps.googleapis.com/maps/api/staticmap?center=Albany,+NY&zoom=13&scale=1&size=600x300&maptype=roadmap&format=png&visual_refresh=true" alt="Google Map of Albany, NY">
+    </a>
+    ```
+
++ Twitter screenshots
+    Similar to map screenshots, this concept allows you to dynamically embed a Twitter screenshot instead of the live feed. [Tweetpik](https://tweetpik.com/) is one of the tools that can be used to take screenshots of tweets. Tweetpik API accepts the URL of the tweet and returns an image with its contents. The API also accepts parameters to customize the background, colors, borders, and dimensions of the image.
```

Comment:
```suggestion
**Twitter screenshots**: Similar to map screenshots, this concept allows you to dynamically embed a Twitter screenshot instead of the live feed. [Tweetpik](https://tweetpik.com/) is one of the tools that can be used to take screenshots of tweets. Tweetpik API accepts the URL of the tweet and returns an image with its contents. The API also accepts parameters to customize the background, colors, borders, and dimensions of the image.
```